projects/packages/changelogger/src/CommandLoader.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Compatibility stub for Symfony 6 changes.
  *
- * Symfony 6 (for PHP 8.0+) added return type hints to its interface. But we still support PHP 5.6, which doesn't recognize that syntax.
+ * Symfony 6 (for PHP 8.0+) added return type hints to its interface. But we still support PHP 7.0, which doesn't recognize that syntax.
```

Comment:
See, this isn't even true. PHP 7.0 does recognize the type hints.

Really at this point we don't need the stub anymore, we can just copy the code from the php7 dir here.

Reply:
That makes sense. Is c59548eb338e3f76d0a4473d5f2158e4d62ca8a5 what you had in mind?

Reply:
Yes. Except since it's a major version bump anyway, let's just delete CommandLoaderBase.php.

Reply:
Done in dfd024f1218c576e931fe46324c7e3e034123132